server/src/game.ts

```diff
@@ -194,272 +195,299 @@ export class Game {
         this.allowJoin = true;
 
         // Start the tick loop
-        this.tick(30);
+        this.tickDynamically(3);
```

Comment:
`this.tick(30)` meant tick at regular 30ms intervals; `this.tickDynamically(3)` means tick as quickly as possible, waiting for 3ms after every tick (to allow event-loop to breathe)